pkg/apps/application-list.jsx

```diff
@@ -117,14 +120,35 @@ export const ApplicationList = ({ metainfo_db, appProgress, appProgressTitle, ac
         }
     }
 
+    async function check_missing_data(packages) {
+        try {
+            const missing = await check_uninstalled_packages(packages);
+            setDataPackagesInstalled(missing.size === 0);
+        } catch (e) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19281-20231012-085302-a6f6f8f0-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)